examples/reshare/src/dkg/actor.rs

```diff
@@ -186,13 +193,18 @@ where
                 (0, initial_public, initial_share)
             };
         let all_participants = Self::collect_all(&active_participants, &inactive_participants);
-        let (active_participants, inactive_participants) = Self::select_participants(
+        let (active_participants, mut inactive_participants) = Self::select_participants(
```

Comment:
nit: I find "inactive" participants pretty confusing...I wonder if we could just use `dealers/players`?